pkg/backends/fake/fake.go

```diff
@@ -0,0 +1,117 @@
+package fake
+
+import (
+	"context"
+
+	"go.uber.org/zap"
+	batchV1 "k8s.io/api/batch/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	loadTestV1 "github.com/hellofresh/kangal/pkg/kubernetes/apis/loadtest/v1"
+)
+
+var (
+	sleepImage = "alpine"
+	imageTag   = "latest"
```

Comment:
I would stick to one of the latest versions, maybe major, but not latest

Reply:
Not super critical tbh, put I pinned.